docs/nut-command/database/database-import.md

```diff
@@ -23,25 +17,41 @@ file.
 
 ## Options
 
-| Option | Description |
-|--------|-------------|
-| -f, --file=FILE | A YAML or JSON file to use for import data. Must end with .yml, .yaml or .json (multiple values allowed)
-
+| Option                        | Description |
+|-------------------------------|-------------|
+| `-f, --file[=FILE]`           | A YAML or JSON file to use for import data. Must end with .yml, .yaml or .json
+| `-d, --directory[=DIRECTORY]` | A source directory. YAML or JSON files generated by the export command will be parsed in order
+|     `--overwrite`             | Allow overwriting of existing records
+| `-h, --help`                  | Display this help message
+| `-q, --quiet`                 | Do not output any message
+| `-V, --version`               | Display this application version
+|     `--ansi`                  | Force ANSI output
+|     `--no-ansi`               | Disable ANSI output
+| `-n, --no-interaction`        | Do not ask any interactive question
+| `-v|vv|vvv, --verbose`        | Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
 
 ## Example
 
 ### Importing records
 
-
 ```bash
-$ php ./app/nut database:import --file=my-site-export.json
+$ php ./app/nut database:import --file=my-site-export.json --overwrite
+
+ [WARNING] This command may overwrite or replace data in the current database.
 
+ Are you sure you want to continue with the import (yes/no) [yes]:
+ > y
 
-WARNING THIS IS AN EXPERIMENTAL FEATURE
+ ! [NOTE] Processing:
+ !        - Export date: 2017-11-22T17:14:28+01:00
+ !        - Bolt version: 3.4.1
```

Comment:
Should we replace the major/minor part with the version macro?

Reply:
Nah, I don't think so.. The people who will use this can read it just fine, i assume. 

Also, this is just the current output, so it is what it is. ;-)

Reply:
Yeah, but in two versions time we'll get someone PRing a change to that to "look up to date" and that just creates merge hell